sandy/zam.py

```diff
@@ -510,6 +524,9 @@ def expand_za(za, method="nndc", meta=0):
         if a >= 300:
             m = 1
             a = a - 300 - m*100
+        while a > 3*z:
+            m += 1
+            a -= 100
```

Comment:
I don't really like this solution,  since, as you said, it is not global. I would just raise a warning unless we have a real equation